Tevinthuku/politico-react-manual-react-config
# List all parties

### What does this PR do?

This PR sets up the view all parties view

### Tasks to be accomplished

- [x] - setup layout navigation
- [x] - create listparties container
- [x] - setup /listparties endpoint
- [x] - setup parties endpoint with partiesreducer

### How can this be manually tested?

- Clone the repo and checkout to `ch/partieslist`
- Start the dev server & go to `/listparties` and you will see a list of all parties.
